rng/config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="file:/local/repos/GH/sknorr/dietrich/rng/conversion.rnc" type="application/relax-ng-compact-syntax"?>
```

Comment:
I think you shouldn't commit that line. :)

Reply:
Right! 👍